src/js/services/storageService.js

```diff
@@ -119,6 +131,123 @@ angular.module('copayApp.services')
         }
       });
     };
+
+    function _upgrade_bitpayCardCredentials(key, network, cb) {
+      key += '-' + network;
+      storage.get(key, function(err, data) {
+        if (err) return cb(err);
+        if (data != null) {
+          // Needs upgrade
+          _01_setAppIdentity(network, data, function(err) {
+            if (err) return cb(err);
+            storage.remove(key, function() {
+              cb(null, 'replaced with \'appIdentity\'');
+            });
+          });
+        } else {
+          cb();
+        }
+      });
+    };
```

Comment:
I think it would be safer that when we upgrade a key, we ALWAYS use a new key for the upgraded data,
as it is done for bitpayCardCredentials and setBitpayDebitCards.

Could you change the new of the key of the upgraded value of `bitpayAccounts` ?